.prow.yaml

```diff
@@ -217,6 +217,28 @@ presubmits:
               memory: 1Gi
               cpu: 500m
 
+  - name: pull-machine-controller-e2e-aws-arm
```

Comment:
ARM support has been added in [v1.21.0](https://github.com/kubermatic/machine-controller/releases/tag/v1.21.0). Does it make sense to test ARM machines with an unsupported version of MC?

Reply:
No, that's just a relic of cherrypick from far future :D 
Also there is some RHEL fix is missing and it keep failing :(